spec.md

```diff
@@ -913,10 +918,16 @@ reg myreg: SInt, myclock
 ; ...
 ```
 
-Optionally, for the purposes of circuit initialization, a register can be
-declared with a reset signal and value. In the following example,
-`myreg`{.firrtl} is assigned the value `myinit`{.firrtl} when the signal
-`myreset`{.firrtl} is high.
+A register may be declared with a reset signal and value. The register's value 
```

Comment:
Mega nit: inconsistent use of single and double space after period in this paragraph.